### ISSUE TITLE
2017 Disclosure Adding/Replacement

### ISSUE THREAD

**Issue by willdwyer** [willdwyer/bcbsmaissuestracker#369]
Hi Michelle,

For 2017 plans, please replace all respective current versions of disclosures with the ones attached.  Let me know if you have any questions.

[55-1339_288_PPO_Blue Options v.5_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366519/55-1339_288_PPO_Blue.Options.v.5_2017.pdf)
[Grandfather_Status_Disclosure_ASC_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366518/Grandfather_Status_Disclosure_ASC_2017.pdf)
[55-1253_288_Preferred Blue Options v.5_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366521/55-1253_288_Preferred.Blue.Options.v.5_2017.pdf)
[55-1252_288_HMO Blue NE Options v.5_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366522/55-1252_288_HMO.Blue.NE.Options.v.5_2017.pdf)
[55-1251_288_HMO Blue Options v.5_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366517/55-1251_288_HMO.Blue.Options.v.5_2017.pdf)
[55-0648_MCC_Does Not Meet_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366520/55-0648_MCC_Does.Not.Meet_2017.pdf)
[55-0647_MCC_Meets_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366525/55-0647_MCC_Meets_2017.pdf)
[55-0646_Chapter 61_PPO_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366523/55-0646_Chapter.61_PPO_2017.pdf)
[55-0645_Chapter 61_HMO_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366524/55-0645_Chapter.61_HMO_2017.pdf)
[55-0613_Pediatric Dental (BCBSMA plans)_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366526/55-0613_Pediatric.Dental.BCBSMA.plans._2017.pdf)
[55-0532_HCCS_HMO_HMONE_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366527/55-0532_HCCS_HMO_HMONE_2017.pdf)
[55-0531_HCCS_PPO_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366529/55-0531_HCCS_PPO_2017.pdf)
[55-0353_Grandfather Status_Insured_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366528/55-0353_Grandfather.Status_Insured_2017.pdf)
[55-1362_HMO_Blue_Select_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366531/55-1362_HMO_Blue_Select_2017.pdf)
[55-0249 limited regional tiered provider_Consumer Guide.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/366530/55-0249.limited.regional.tiered.provider_Consumer.Guide.pdf)

**Comment by michparrish** [comment]
Hi Will,

Do you need these in addition to the current versions, in case you are generating any 2016 docs? I don't want to wipe those out with these 2017 versions. I wanted to be sure. If you want/need both, I need to set up the proper mapping to point to Plan Year attribute = 2017 vs 2016 for all as well.
Please confirm.
thanks!

**Comment by willdwyer** [comment]
We would need these in addition to the current versions.  The current versions would apply to 2016 jobs, while the disclosures attached here would apply to 2017 jobs.

**Comment by michparrish** [comment]
ok thank you - let me set up in pclient5 and get the mapping set up too. You can test out both to make sure you're getting the right copies.

**Comment by michparrish** [comment]
Ready for testing - please let me know if you run into any issues.

**Comment by michparrish** [comment]
...in pclient5

**Comment by willdwyer** [comment]
I'm seeing this error when trying to generate (have seen it 3 times).

![screen shot 2016-07-19 at 2 59 53 pm](https://cloud.githubusercontent.com/assets/13453008/16962856/7cc46e9a-4dc1-11e6-9a46-e3803de2d65a.png)

**Comment by michparrish** [comment]
Will,
I'm looking into this now - I retested and experienced the same, both with and without disclosures, so I don't believe it's related to those updates. I'll report back when I hear from QA. 
Thank you for your patience.

**Comment by michparrish** [comment]
Hi Will,
Please give it another try - I've been able to successfully generate 2016 and 2017 SBCs with the proper year's Disclosure stitched to it. Please let me know how your testing goes.
Thanks!

**Comment by willdwyer** [comment]
Looks good, for the most part.  However, one issue I noticed is that the plan 900049HR_Preferred_Blue_Ded_500_80-0259HCCSSG1-1-17 generates with the 55-0645 disclosure.  It should generate with the 55-0646 disclosure.  The requirements for this were depicted in #365.  Please see screenshot and fix.
<img width="629" alt="screen shot 2016-07-22 at 10 50 37 am" src="https://cloud.githubusercontent.com/assets/13453008/17061101/e6024068-4ffa-11e6-96d5-9f5f9ba6f74f.png">

**Comment by michparrish** [comment]
Hi Will, 
Looks like the mapping and attributes on the plan are all correct. If you click on the hyperlink above for 55-0646 and open the doc, you'll notice that the code in the bottom of the doc is 55-0645. Is that not the right document?
Please upload the proper doc for this code and I'll switch it out in pclient5.
Thanks!

**Comment by willdwyer** [comment]
Hi Michelle, we realized we are missing the 55-0646 disclosure, and at some point the 55-0645 disclosure was duplicated in its place.  This is an internal issue that we are assessing on our end.  We will leave this assigned to us for now.

**Comment by willdwyer** [comment]
Hi Michelle,

We have updated both.  Please see both 55-0645 and 55-0646 below.
[55-0646_Chapter 61_PPO_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/394236/55-0646_Chapter.61_PPO_2017.pdf)
[55-0645_Chapter 61_HMO_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/394237/55-0645_Chapter.61_HMO_2017.pdf)

**Comment by michparrish** [comment]
Ok - new versions have been uploaded. Please feel free to continue your testing.
Thanks!

**Comment by willdwyer** [comment]
Looks good.  Please move to production.

**Comment by michparrish** [comment]
This is all set in Prod. Please let me know if there are any issues - there may be one thing I have to do (remove prior versions of the mapping instead of adding a new version). It's an easy change but didn't know if I needed to make it in Prod.

**Comment by willdwyer** [comment]
Hi Michelle,

Did HighRoads re-add the mapping for this?  (I was out all of last week)

**Comment by willdwyer** [comment]
Hello,

For 2017 connector plans, this Pediatric Dental disclosure (55-0613) should replace the 55-0614 disclosure when generated.  See attachment.

[55-0613_Pediatric Dental (BCBSMA plans)_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/425855/55-0613_Pediatric.Dental.BCBSMA.plans._2017.pdf)

**Comment by michparrish** [comment]
This mapping should already be in place - I have that document, from the first full batch of 2017 docs. Do you want me to update the mapping in Prod so you can test all of them out?

**Comment by willdwyer** [comment]
Yes, please update the mapping in Prod so we can test all of them.

As far as the 55-0613 document is concerned, this disclosure inclusion already existed for small group plans, but the change that will be incurring for 2017 is that this will also be generated for connector plans as well (i.e. for 2016 the disclosure for connector plans 55-0614 not 55-0613).

Let me know if you have any questions on this.

**Comment by michparrish** [comment]
no questions - apparently that message had been provided at some point b/c that's how the mapping is set up right now. I've uploaded the mapping doc into Prod. Let me know if there are any issues!

**Comment by willdwyer** [comment]
All looks good.  As mentioned, we have a new ACA Discrimination disclosure coming that will be added to ALL 2017 standard templates.  As a result, this will be left in Monitoring/On Hold status until that comes about.

Either way, this is all set for now.

**Comment by michparrish** [comment]
excellent! Thanks!

**Comment by willdwyer** [comment]
Hi,

Note that we will be receiving the new ACA discrimination disclosures today that must be appended to ALL 2017 standard templates.  This will also be an urgent matter, as we must have these complete and sent out by 9/1, so please prepare accordingly.

**Comment by michparrish** [comment]
Thanks for the head's up. Shouldn't be an issue. Please provide the document and we can map to attribute Plan Year = 2017, therefore all docs with Plan Year = 2017 should have this document. Also, please provide the order in which you need this to show.

**Comment by willdwyer** [comment]
Michelle,

Is it easier for you to attach 1 disclosure, and then a second one after that?  Or, is it easier to just do both at once?

Reason I ask - we have 1 disclosure ready, but are still waiting on the other.

**Comment by michparrish** [comment]
Would probably be easiest to do them both at the same time. Thanks!

**Comment by willdwyer** [comment]
Hi Michelle,

All 3 disclosures are attached.  Instructions are as follows:

For **ALL** 2017 plans, the 55-1487 disclosure should be attached.
For 2017 **connector** plans, the 55-1491 disclosure should be attached (after the 55-1487 disclosure).
For 2017 **non-connector** plans (any market segment that is not connector), the 55-1493 disclosure should be attached (after the 55-1487 disclosure).  BCBSMA needs to aggressively meet deadline dates of 9/1 and 9/7 for sending these out, so please complete this ASAP.

[55-1493_ACA Language Translation Services_NonQHP.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/448336/55-1493_ACA.Language.Translation.Services_NonQHP.pdf)
[55-1491_ACA-Translation Services_Connector_QHP.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/448334/55-1491_ACA-Translation.Services_Connector_QHP.pdf)
[55-1487_ACA_NonDiscrimination_Notice_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/448335/55-1487_ACA_NonDiscrimination_Notice_2017.pdf)

**Comment by michparrish** [comment]
Just to confirm, for the 55-1493, that should be mapped to Small Group, Large 51-99, Large 100+ AND Student group market segments?

**Comment by michparrish** [comment]
This is all set - everything is uploaded and mapped in pclient5. Test it out and I'll move everything to Prod for you as soon as possible.

**Comment by willdwyer** [comment]
Hi Michelle,

We may have to replace the 55-1493 and 55-1491 as we just got edits from our legal department.  These would just replace the 2 disclosures that are already in PCLIENT5.  More to come on this.

Thanks for your help.

**Comment by michparrish** [comment]
no problem. Do you want to test out the mapping/results now and we can move all of that into Prod, but then just replace those docs in prod when you get them? that way we can be set up to go right when you get them over to me.

**Comment by willdwyer** [comment]
Yes, I am doing that now.

**Comment by willdwyer** [comment]
Hi Michelle,

Please use these disclosures.  These will replace the other 2.  As mentioned, instructions are as follows:

For ALL 2017 plans, the 55-1487 disclosure should be attached.
For 2017 connector plans, the 55-1491 disclosure should be attached (after the 55-1487 disclosure).
For 2017 non-connector plans (any market segment that is not connector), the 55-1493 disclosure should be attached (after the 55-1487 disclosure). BCBSMA needs to aggressively meet deadline dates of 9/1 and 9/7 for sending these out, so please complete this ASAP.
[55-1493_ACA Language Translation Services_NonQHP.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/457060/55-1493_ACA.Language.Translation.Services_NonQHP.pdf)
[55-1491_ACA-Translation Services_Connector_QHP.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/457059/55-1491_ACA-Translation.Services_Connector_QHP.pdf)

**Comment by michparrish** [comment]
new docs have been uploaded into pclient5 - check it out and if no issues, we can move into Prod (I just haven't moved the new mapping into Prod)

**Comment by michparrish** [comment]
new docs are ready and waiting in Prod as well - as soon as you test successfully, I'll move the mapping over to Prod and we should be good to go

**Comment by willdwyer** [comment]
Michelle, I tested on PCLIENT5, and these disclosures are not at the end.  Please do so.  (should be 55-1487, followed by 55-1493 or 55-1491)

**Comment by michparrish** [comment]
Will - I just tested as well and the docs show up. What were your steps?
I searched for a 2017 standard plan (any) and went straight to Plan Documents to generate docs. Selected Final, Disclosures = Yes, PDF and Submitted.

**Comment by willdwyer** [comment]
No, Michelle - I am referring to the _order_ of the disclosures.  55-1487 and 55-1493 or 55-1491 should be the last of all disclosures on generation.

**Comment by willdwyer** [comment]
Please let us know once fixed, and we will re-test.

**Comment by michparrish** [comment]
ah my bad. Yes, it wasn't clearly stated at first so I put them at the beginning. I'll change.

**Comment by michparrish** [comment]
mapping was updated - have another look please

**Comment by willdwyer** [comment]
@cfieldsHR @randylstillman 

If Michelle is unavailable, can one of you two assist?  This is of top priority for us, as we have a deadline of 9/7 for which we must generate all insured SBCs in compliance.

**Comment by willdwyer** [comment]
OK, thanks Michelle.  Will re-test now.

**Comment by willdwyer** [comment]
Looks good.  Please move to production.

**Comment by michparrish** [comment]
all set!

**Comment by willdwyer** [comment]
Hi Michelle,

Please add the 55-1487 and 55-1491/55-1493 disclosure mapping to 2016 SBC generation going forward as well.

**Comment by michparrish** [comment]
Hi there,
Updated the mapping in pclient5 if you want to check it out.

**Comment by willdwyer** [comment]
Looks good.

Please move to production.

**Comment by michparrish** [comment]
all set!

**Comment by mattrimer** [comment]
Hi Michelle,

Please add 55-1339_288_PPO_Blue Options v.5_2017 to all PPO Blue Options Plans:

MM (51-99) ASC
900168
900169
900170

Large (100+) ASC
900342
900343
900344

Thanks
[55-1339_288_PPO_Blue Options v.5_2017.pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/481050/55-1339_288_PPO_Blue.Options.v.5_2017.pdf)

**Comment by michparrish** [comment]
Hi Matt,
Looks like this Disclosure doc is already ready for use? I don't actually attach them to the plans - they are attached via mapping when the SBC is generated and 'Disclosures' = Yes on the Doc Gen screen. You should be able to generate them.

**Comment by mattrimer** [comment]
Right I understand that--I should have explained this more. When trying to generate documents from these 6 plans, all disclosures attach correctly, however the disclosure previously mentioned (55-1339) is missing. It should appear right after 55-0647.

**Comment by michparrish** [comment]
hmm - ok, it's in the mapping. Do those plans have the following attributes?
Product Group - PPO Blue Options
Provider Network Tier - 4
Plan Year - 2017

It looks like 900168 and 169 have 'PPO Blue (ASC)' as the Product Group - if that is the case for all of them, I can add that to the mapping as well.

**Comment by mattrimer** [comment]
The Provider Network Tier and Plan Year attributes are correct, so I suspect in has to do with the Product Group. Can you update the product group to reflect "PPO Blue Options"?

Hopefully that'll work. Thanks!

**Comment by michparrish** [comment]
Are they ASC plans, though? I can either update the mapping or you can update the attribute. I am of the understanding that I shouldn't touch your Prod plan data. It's an easy update for you, though!

**Comment by mattrimer** [comment]
PPO Blue Options is not a selectable Product Group Type attribute, so I think you'll need to update the mapping so that this disclosure will link to PPO Blue ASC, 4-tier, 2017.

To answer your question, these plans are ASC, however there are no insured equivalents. I think we had changed that attribute name from "PPO Blue Options" to "PPO Blue ASC" because a new, non-options, PPO Blue plan was added for 2017. This new plan should not have the disclosure, but it shouldn't be affected by your update as it is 2-tier.

**Comment by michparrish** [comment]
Updated in pclient5 - check it out and if it looks good, I'll move the new mapping to Prod

**Comment by mattrimer** [comment]
Hi Michelle,

I'm receiving the following error message:

![screen shot 2016-09-20 at 1 41 09 pm](https://cloud.githubusercontent.com/assets/18705500/18681899/001c0fe4-7f38-11e6-86f9-f20adb549258.png)

Please advise.

**Comment by michparrish** [comment]
please try again - I removed the old line in case that was causing the issue

**Comment by mattrimer** [comment]
I retested and I'm still getting the same error

**Comment by michparrish** [comment]
ugh, sorry - there was an extra, blank row. Please try again!!

**Comment by mattrimer** [comment]
Looks good--please push to production.

Thanks!

**Comment by michparrish** [comment]
all set in Prod!

**Comment by mattrimer** [comment]
Looks great-- thanks Michelle!

We'll keep this Issue open for now in case we have any other disclosure updates.

**Comment by willdwyer** [comment]
Hi Michelle,

The attached 55-0645 disclosure should replace the current existing disclosure for plans with the following attributes:
Plan Year - 2017
Market Segment - Small Group 1-50
Product Type - HMO; HMO-TYPE
Product Identifier - HCCS; Select Limited Network; Tiered Network

_NOTE 1:_ this will NOT replace the 55-0645 disclosure for any plans that have an N/A attribute under Product Identifier
_NOTE 2:_ this will NOT replace any of the plans that have the 55-0646 (PPO) disclosure

[55-0645_Chapter 61_HMO_2017[1].pdf](https://github.com/willdwyer/bcbsmaissuestracker/files/496254/55-0645_Chapter.61_HMO_2017.1.pdf)

**Comment by michparrish** [comment]
Document replaced in pclient5 - please test and confirm and let me know if I can move into Prod

**Comment by willdwyer** [comment]
Error.  See attached.
![screen shot 2016-09-29 at 3 39 07 pm](https://cloud.githubusercontent.com/assets/13453008/18969369/e3da2b8e-865a-11e6-9a0f-dccaf46feafc.png)

**Comment by michparrish** [comment]
please try again - had to rename the doc in the folder.

**Comment by willdwyer** [comment]
Looks good.  please move to production

**Comment by michparrish** [comment]
all set
